src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,71 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+import { Paragraph } from '../Paragraph';
+
+const Notification = ({ toast, message, body, status, onClose }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  const { icon: CloseIcon } = theme.notification.button;
+  const { icon: StatusIcon, color } = theme.notification[status];
+  const iconColor = theme.notification.icon.color;
+
+  let content = (
+    <Box direction="row">
+      <Box
+        {...theme.notification.iconContainer}
+        background={color}
+        justify="center"
+      >
+        <StatusIcon color={iconColor} />
+      </Box>
+      <Box
+        {...theme.notification.textContainer}
+        align="start"
+        direction="row"
+        justify="between"
+        fill
+      >
+        <Box>
+          <Text {...theme.notification.messageText}>{message}</Text>
+          {body && (
+            <Paragraph {...theme.notification.bodyText}>{body}</Paragraph>
```

Comment:
I'm wondering about if someone wanted to do something like this in the notification body where an anchor is in the text:
```
If you have plans to increase utilization in the near future, additional
      capacity can be added by selecting the <Anchor label="Order Capacity" />
      action from the action menu.
```

Would it be better for Notification to accept children, where the children are placed in the paragraph? I've found having long strings/combo of string + JSX doesn't always feel as clean as a prop.

implementation would be something like

```
<Notification toast message="My Notification">
   Here is my body text with <Anchor label="anchor" />.
</Notification>

Reply:
I was inclined to leave children for all text content, allowing mixing strings and JSX, but then not allowing the `message` or `body` props in that case.